isis/src/control/objs/ControlNet/ControlNet.cpp

```diff
@@ -1086,11 +852,17 @@ namespace Isis {
    * @param serialNumber the cube serial number to validate
    */
   void ControlNet::ValidateSerialNumber(QString serialNumber) const {
-    if (!cubeGraphNodes->contains(serialNumber)) {
-      IString msg = "Cube Serial Number [" + serialNumber + "] not found in "
-          "the network";
-      throw IException(IException::Programmer, msg, _FILEINFO_);
+//  if (!cubeGraphNodes->contains(serialNumber)) {
+//    IString msg = "Cube Serial Number [" + serialNumber + "] not found in "
+//        "the network";
+//    throw IException(IException::Programmer, msg, _FILEINFO_);
+//  }
+    if (!m_vertexMap.contains(serialNumber)) {
+          IString msg = "Cube Serial Number [" + serialNumber + "] not found in "
+              "the network";
+          throw IException(IException::Programmer, msg, _FILEINFO_);
```

Comment:
We should consider changing this to just return a bool instead of throwing an exception.

Reply:
I did change this to returning a bool -- here are some followup questions: maybe or maybe not for the standup: 

* GetMeasuresInCube -- if can't validate SN, return an empty list or throw exception? 
* GetValidMeasuresInCube -- if can't validate SN, return an empty list or throw exception? 
* DeleteMeasuresWithID -- if we can't validate the SN, do we delete no measures, or throw an exception.